tests/cypress/e2e/rmrk-simple-mint.cy.ts

```diff
@@ -35,13 +35,8 @@ describe('simple mint in rmrk', () => {
     // description
     cy.get('[data-cy="input-description"]').type('this is some description')
 
-    // check edition
```

Comment:
@prachi00 could you adjust it instead of removing this section?

```
assertexpected <input.o-input> to have value '3', but the value was '1'
```

Reply:
we do not have `.control.minus button` this anymore though, how do we rewrite the minus button part then?

Reply:
you dont need to find that arrow. just use `type` https://docs.cypress.io/api/commands/type#InputTextarea that was native input